docs/process.md

```diff
@@ -1,3 +1,31 @@
+Development Processes
+---------------------
+
+Landing PRs
+===========
+
+ * Even after the code of a PR is approved, it should only be landed if the
+   CI on github is green, or the failures are known intermittent things.
```

Comment:
we could be more specific about "intermittent things", e.g. "known to be unrelated to the PR itself".